### PR TITLE
Merge devel on main

### PR DESCRIPTION
**Pull request by Alex-Izquierdo**
Merge devel on main and keep main as the base branch for the latest changes.

### CODE REVIEW

**Comment by Alex-Izquierdo** [comment]
The idea is rebase the history because the branch contains different additions. 
I can rebase the noisy commits if you prefer. @ansible/eda-maintainers

File: pyproject.toml [hunk 1]
```diff
@@ -42,15 +42,13 @@ cryptography = "*"
...
-# Experimental LDAP Integration https://issues.redhat.com/browse/AAP-16938
-# We are temporarily updating it to use latest from devel branch to keep consistency accross
-# other AAP components
 django-ansible-base = { git = "https://github.com/ansible/django-ansible-base.git", tag = "2024.1.31", extras = [
     "authentication",
     "channel_auth",
 ] }
 jinja2 = "*"
 django-split-settings = "^1.2.0"
+ansible-core = "^2.16.2"
```
Comment (mkanoor) [inline pyproject.toml:51]: @Alex-Izquierdo Should this come off

**Review by Dostonbek1 — DISMISSED** [review]
lgtm

File: pyproject.toml (hunk 1 above)
Comment (Alex-Izquierdo) [reply pyproject.toml:51]: The line belongs to devel. Removing or changing that feature should be done in a different PR.

**Review by mkanoor — DISMISSED** [review]

**Review by hsong-rh — DISMISSED** [review]

**Review by bzwei — DISMISSED** [review]

**Review by ttuffin — APPROVED** [review]

**Pull request merged**